scripts/vendorsingle.sh

```
@@ -1,23 +1,35 @@
 #!/usr/bin/env bash
 echo $1
 v=$(basename $1)
-mib=$(find $1 -type f  | sed 's|$1/||g')
+mib=$(find $1 -type f ! -name '.*'  | sed 's|$1/||g')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)